data/Arts2023Longitudinal.yaml

```diff
@@ -0,0 +1,3855 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal and quantitative fecal shedding dynamics of SARS-CoV-2, pepper
+  mild mottle virus, and crAssphage
+doi: 10.1128/msphere.00132-23
+description: >
+  The authors present longitudinal, quantitative fecal shedding data for SARS-CoV-2
+  RNA, pepper mild mottle virus (PMMoV) RNA, and crAss-like phage (crAssphage) DNA
+  from 48 COVID-19 patients. Abundances were quantified using (RT)-ddPCR assays targeting
+  the N and ORF1a genes. The data were contributed by the corresponding author.
+analytes:
+  stool_SARSCoV2_N:
+    description: >
+      Concentration of RNA of the N gene quantified using (RT)-ddPCR in stool samples.
+      The concentration were quantified in gene copies per dry weight of stool. The
+      limit of blank (LOB), determined as the upper 95% confidence limit of the negative
+      extraction control, ranged from 11.2 to 1,550 gc/mg-dry weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: N gene
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_SARSCoV2_ORF1a:
+    description: >
+      Concentration of RNA of the ORF1a gene quantified using (RT)-ddPCR in stool
+      samples. The concentration were quantified in gene copies per dry weight of
+      stool.The limit of blank (LOB), determined as the upper 95% confidence limit
+      of the negative extraction control, ranged from 11.2 to 1,550 gc/mg-dry
+      weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: ORF1a
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_PMMoV:
+    description: >
+      Concentration of PMMoV RNA was measured by ddPCR on the same day as the SARS-CoV-2.
+      The concentration were quantified in gene copies per dry weight of stool.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 400
+    specimen: stool
+    biomarker: PMMoV
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_crAssphage:
+    description: >
+      Concentration of CrAssphage DNA was measured by qPCR. LOQ if not quantifiable.
```

Comment:
What does "LOQ if not quantifiable." mean here?

Reply:
The reported number is either the measured concentration of crAssphage or the Limit of Quantification (LOQ) if the concentration wasn't quantifiable. This information comes from the supplementary data, where the description for the crAss_conc column is 'measured concentration of crAssphage, or the LOQ if not quantifiable.' However, I didn't find any descriptions about the LOQ of crAssphage concentration in the main paper. I'll update the description for clarity. Thanks for pointing that out!

Reply:
What do you think of this logic to get the values (using variable names from the Excel supplement)?

```python
if crAss_quant:
    value = crAss_conc
elif crAss_det:
    value = "positive"
else:
    value = "negative"
```

Reply:
I like this logic! Previously, I used a similar approach for N_conc (gc/mg-dw), but as we discussed, preserving more information for numeric values below the LOB is better. However, the case with crAss is slightly different because, when the concentration is not quantifiable, the LOQ value is used. Based on my observations, the crAss_conc values vary even when crAss_det is false. Do we want to keep this information?

Reply:
Yes, good point. Interesting that what the dataset says "or the LOQ if not quantifiable" doesn't actually seem to be the case. Shall we just leave things as is?

Reply:
I agree!

Reply:
I just want to say that you both are working too hard! 😅 I need to cook dinner but will review this first thing tomorrow.